src/fiatExchanges/FiatExchangeCurrency.tsx

```diff
@@ -114,6 +118,7 @@ function FiatExchangeCurrency({ route, navigation }: Props) {
     navigate(Screens.FiatExchangeAmount, {
       currency: selectedCurrency,
       flow: flow === FiatExchangeFlow.CashIn ? CICOFlow.CashIn : CICOFlow.CashOut,
+      network: selectedCurrency === CiCoCurrency.ETH ? Network.Ethereum : Network.Celo,
```

Comment:
a more bug safe way to do this might be to create a mapping from cico currency to network and use type `Record<CiCoCurrency, Network>`, so we get a type error if we try to add a cicocurrency but forget to specify the correct network for this line

Reply:
Ah yeah, I like that idea -- added it in!